nrf_security/doc/drivers.rst

```diff
@@ -0,0 +1,115 @@
+.. _nrf_security_drivers:
+
+nrf_security drivers
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nrf_security module supports multiple enabled PSA drivers at the same time.
+This mechanism is intended to extend the available feature set of hardware-accelerated cryptography or to provide alternative implementations of the PSA Crypto APIs.
+
+Enabling a cryptographic feature or algorithm is done using PSA Crypto API configurations that follows the format ``PSA_WANT_ALG_XXXX``.
+
+Enabling more than one PSA driver may add support for additional key sizes or modes of operation. 
+
+It is possible to disable specific features on PSA driver level to optimize the code size.
+
+The nrf_security module supports the following PSA drivers:
+
+* Arm CryptoCell cc3xx
+* nrf_oberon binary library
+
+.. note::
+   Note that whenever this documentation mentions 'original' Mbed TLS, it refers to the open-source `Arm Mbed TLS project`_, not the customized version available in Zephyr.
+   There is an option to utilize a 'builtin' driver, which corresponds to software implemented cryptography from the 'original' Mbed TLS deliverables. 
+   This is provided to ensure that the cryptographic toolbox supports all requested features.
+
+.. _nrf_security_drivers_cc3xx:
+
+Arm CryptoCell cc3xx driver
+***************************
+
+The Arm CryptoCell cc3xx driver is a runtime library that provides hardware-accelerated cryptography using the Arm CryptoCell cc310/cc312 hardware.
```

Comment:
The interesting part is that this library is not delivered in source form.

So

"is a closed-source binary library that provides"

Reply:
Fixed (part of next force-push)

Reply:
let's use the same phrasing as oberon